tools/docker/.env

```diff
@@ -8,7 +8,6 @@ CHAINLINK_PGPASSWORD=node
 ETH_CHAIN_ID=34055
 ETH_URL=ws://devnet:8546
 EXPLORER_URL=ws://explorer:3001
-LOG_LEVEL=info
```

Comment:
on second thought I think maybe we leave this in - spoke offline

Reply:
reverted~